### PR TITLE
Users/shrverm/upgrade npm tasks

### PR DESCRIPTION
**Pull request by shadargee1982**
**Task name**: NpmV0, NpmV1

**Description**: Reference new versions of sharing packages, that support retries

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** https://portal.microsofticm.com/imp/v3/incidents/details/214296589/home

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/NpmV1/npm.ts [hunk 1]
```diff
@@ -77,13 +77,13 @@ async function _logNpmStartupVariables(packagingLocation: pkgLocationUtils.Packa
...
         const npmTelem = {
             'command': command,
-            'verbose': tl.getInput(NpmTaskInput.Verbose),
-            'customRegistry': tl.getInput(NpmTaskInput.CustomRegistry),
-            'customFeed': tl.getInput(NpmTaskInput.CustomFeed),
-            'customEndpoint': tl.getInput(NpmTaskInput.CustomEndpoint),
-            'publishRegistry': tl.getInput(NpmTaskInput.PublishRegistry),
-            'publishFeed': tl.getInput(NpmTaskInput.PublishFeed),
-            'publishEndpoint': tl.getInput(NpmTaskInput.PublishEndpoint),
+            'verbose': tl.getInput(NpmTaskInput.Verbose) || null,
```
Comment (zhenghao104) [inline Tasks/NpmV1/npm.ts:80]: What is `|| null` for, and why it wasnt needed before?

Comment (shadargee1982) [reply Tasks/NpmV1/npm.ts:80]: it was a change that came in from upgraded task lib.  tl.getInput could return undefined, so adding `|| null` will ensure proper value handling.

Comment (zhenghao104) [reply Tasks/NpmV1/npm.ts:80]: But line like this dont need a null catch?

https://github.com/microsoft/azure-pipelines-tasks/blob/575b87434af3198b43541f1ea233825e7d410ec8/Tasks/NpmV0/npmtask.ts#L21

Comment (shadargee1982) [reply Tasks/NpmV1/npm.ts:80]: i used it sparingly in the places that I thought were not as fault-tolerant.  At least in the example here...if the command is undefined, the task would error out, making it easy to detect.

**Review by aasim — APPROVED** [review]

**Pull request merged**